torch/ao/quantization/_dbr/qconfig_dict_utils.py

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, Any
+
+import torch
+
+TYPE_TO_REPLACEMENT_TYPE = {
+    torch.add: torch.Tensor.add,
+    torch.Tensor.add_: torch.Tensor.add,
```

Comment:
do we have `torch.Tensor.add` as well?

Reply:
The user code can have any variant of add, but `torch.Tensor.add` is what we see inside the `__torch_function__` dispatch.  DBR quant only sees the types inside of __torch_function__, so it does not have any knowledge of which variant was in the user's code.